src/transformers/Streamable.js

```diff
@@ -0,0 +1,27 @@
+import { URL } from 'url';
+
+import fetch from 'node-fetch';
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.includes(item));
+
+export const shouldTransform = url => {
```

Comment:
This function can be simplified/made a bit less messy (imo) to

```js
export const shouldTransform = url => {
  const { host, pathname } = new URL(url);
  const trimmedPathName = getTrimmedPathName(pathname);

  return ['streamable.com', 'www.streamable.com'].includes(host) &&
    trimmedPathName.length > 0 &&
    trimmedPathName.length < 3 &&
    (trimmedPathName.length === 1 ||
      includesSomeOfArray(pathname, ['/o/', '/s/']));
};
```

You could just copy over the `getTrimmedPathName` function from the Slides transformer for now